backend/app/src/main/resources/application-dev.yml

```diff
@@ -11,3 +11,16 @@ spring:
     # logging.level.root=DEBUG
     # logging.level.root=ERROR
 
+  security:
+    oauth2:
+      resourceserver:
+        jwt:
+          issuer-uri: https://common-logon-test.hlth.gov.bc.ca/auth/realms/moh_applications
```

Comment:
I will pick this from env variable once I clean the terraform code.

Reply:
Makes total sense.